imgix/urlbuilder.py

```diff
@@ -93,6 +95,19 @@ def create_url(self, path, params={}, opts={}):
         else:
             domain = self._domains[0]
 
+        regex = re.compile(
+                r'^(?:[a-z\d\-_]{1,62}\.){0,125}'
+                r'(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)'
+                r'[a-z\d]{1,63}$'
+        )
+
+        if re.match(regex, domain) is None:
+            warnings.warn(
+                '''\nDomain url does not conform to the required structure.
+                \nCheck to see if the protocol (http/https) was prepended
```

Comment:
This is a little vague, I think. You should make it clear that protocol and path are *not* supposed to be included.

Reply:
Yea definitely agree. I mentioned in a different comment adding guidelines for acceptable domain format in the docs, maybe the error should just point there instead of listing out possible causes.